gnomad_bigquery/bq_table_descriptions.py

```diff
@@ -2,13 +2,29 @@
 
 LEAST_CSQ = '3_prime_UTR_variant'
 
-def get_variants_table_desc(version: int, data_type: str = None):
+def get_variants_table_desc(version: int, data_type: str, split=False, in_gt=False):
     dtype = '' if data_type is None else f' {data_type}'
-    return f"""This table contains all gnomad v{version} {dtype} variants.
-    Notes:
-    * All variants were split, so multi-allelic sites are represented as multiple rows (one per non-ref allele)
-    * Only VEP transcript annotations are available (non-coding annotations aren't)
-    """
+    if split:
+        if in_gt:
+            return f"""This table contains gnomad v{version} {dtype} variants that are filtered similarly to the
+             genotypes table, optionally filtered by a least consequence term and max AF. 
```

Comment:
The AF wasn't used, was it?

Reply:
(same below)

Reply:
No it wasn't, thats why I have optionally in there but I see that is misleading.  I was trying to make this more general use but it doesn't seem like it will work out.